core/build.sbt

```diff
@@ -2,7 +2,7 @@
 libraryDependencies ++= Seq(
   "io.argonaut"                %% "argonaut"          % "6.2.1",
   "org.typelevel"              %% "cats-free"         % "1.5.0",
-  "org.typelevel"              %% "cats-effect"       % "1.1.0"
+  "org.typelevel"              %% "cats-effect"       % "0.10.1"
```

Comment:
hm what is causing the incompat? the point of the earlier PR I made was to bump this dependency so we could start moving Nelson to Cats 1.0

Reply:
Ahh it is HTTP4S -> FS2 -> cats-effect. OK if this fixes it let's merge when ready and cut a release first since folks are waiting on one, and then once HTTP4S 0.20.0 final comes out which has the new FS2 and cats-effect hotness I'll re-do the change.